Examples/Pass/arc.cvf

```diff
@@ -44,16 +34,12 @@ method drop() {
       {| noCnt() |}
         <free = (true)>;
       {| emp |}
-    } else {
-      {| emp |}
-        dummy = dummy; // skip;
-      {| emp |}
     }
   {| emp |}
 }
 
 view error();
```

Comment:
I don't think it's worth doing this, unless it makes your other commit easier to understand. We'll have to resolve the inconsistencies with the paper version of ARC anyway.